scene/gui/graph_edit.cpp

```diff
@@ -748,25 +746,29 @@ bool GraphEdit::_check_clickable_control(Control *p_control, const Vector2 &pos)
 	}
 }
 
-bool GraphEdit::is_in_hot_zone(const Vector2 &pos, const Vector2 &p_mouse_pos, const Vector2i &p_port_size, bool p_left) {
```

Comment:
Why not keep this function and make this overwritable?
Instead of creating 2 new ones and overwriting those?

Reply:
This is to account for whether the user wants to override only one side of the ports.